File: Jugador.java
package Controller;

import java.util.ArrayList;
import java.util.List;

public class Jugador
    {
        public String nombre;
        public List<Baraja.Carta> reparto;

        public Jugador(String nombre) {
            this.nombre = nombre;
            this.reparto = new ArrayList<>();
        }

        public void recibirCarta(Baraja.Carta carta)
        {
            reparto.add(carta);

        }

        public void mostrarCartas()
        {
            System.out.print(nombre + ": ");
            for (Baraja.Carta carta : reparto) {
                System.out.print(carta + " ");
            }
            System.out.println();
        }
        public int calcularValorCartas() {
            int valorTotal = 0;
            for (Baraja.Carta carta : reparto) {
                valorTotal += carta.valorNumerico;
            }
            return valorTotal;
        }
    }

File: Jugador.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Jugador
{
    public string Nombre { get; set; }
    public List<Baraja.Carta> Reparto { get; set; }

    public Jugador(string nombre)
    {
        Nombre = nombre;
        Reparto = new List<Baraja.Carta>();
    }

    public void RecibirCarta(Baraja.Carta carta)
    {
        Reparto.Add(carta);
    }

    public void MostrarCartas()
    {
        Console.Write(Nombre + ": ");
        foreach (Baraja.Carta carta in Reparto)
        {
            Console.Write(carta + " ");
        }
        Console.WriteLine();
    }

    public int CalcularValorCartas()
    {
        int valorTotal = 0;
        foreach (Baraja.Carta carta in Reparto)
        {
            valorTotal += carta.ValorNumerico;
        }
        return valorTotal;
    }
}